### DISCUSSION TITLE
Adding Playlist editing feature.. (move and remove files from playlist)

### DISCUSSION THREAD

**Discussion by chella1cm** [tomasklaen/uosc#456] [Ideas]
I have been using this [Playlist Manager](https://github.com/jonniek/mpv-playlistmanager), it will be really handy to have these features inside uosc playlist manager. 

Up and down to navigate up and down the playlist,
Left and right to select a file and move it up and down the playlist,
Backspace to delete the selected file..

Would really love to see these features in uosc,..
Thanks for the awesome skin..

**Comment by tomasklaen** [comment]
f039af5

`ctrl+up/down` now moves a selected item up/down, and `del` deletes it.

**Reply by chella1cm** [reply]
Wow.. Thanks man.. just checked it works great..
